packages/tokamak/contracts/deploy-config/holesky-nightly.ts

```diff
@@ -0,0 +1,20 @@
+const config = {
+  numDeployConfirmations: 1,
+  l1BlockTimeSeconds: 15,
+  l2BlockGasLimit: 15_000_000,
+  l2ChainId: 4906,
```

Comment:
Do you have a rule for creating it (l2ChainId of holesky and holesky-nightly)?

Reply:
Oh, I'm sorry. I don't have a rule for chainId. It is just what I want. Should we change the chainId?

Reply:
We used chainid 5050 and 5051 in the goerli. I knew that anyone no use chainid [5052](https://chainlist.org/chain/5052), [5053](https://chainlist.org/chain/5053). If you have better idea, we may use you want.

Reply:
I fixed it. Thank you!